local-proxy/README.md

```diff
@@ -0,0 +1,39 @@
+Flipt Local Proxy
+------------
+
+This lab section's intent is to explore the ways that clients can achieve fast evaluations from Flipt for their feature flags. The use case here is that users would like data locality for their feature flag state, so they do not have to worry about network + application time to get a feature flag evaluation value especially for their most critical application paths.
+
+### S3 Replication
```

Comment:
what if we rename this to `object store` and the folder as well? will go with how we call it in our docs: https://www.flipt.io/docs/configuration/storage#object s3 is just an implementation detail I think